### PR TITLE
Add package for 'biotraj'

### PR DESCRIPTION
**Pull request by padix-key**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by JHKru** [comment]
From https://github.com/biotite-dev/biotite/issues/613
@padix-key 
Sure, you can add me to the list.

**Comment by padix-key** [comment]
@conda-forge/staged-recipes This recipe should be ready for review. I have few question though:
The linter complains that the recipe already exists in `conda-forge`. Probably the reason is that I initially submitted the recipe with a wrong directory name. Does this mean I have to close and reopen this PR, or is there a way to *rerender* it?

**Comment by padix-key** [comment]
@conda-forge/help-python-c

**Comment by ocefpaf** [comment]
> Probably the reason is that I initially submitted the recipe with a wrong directory name.

The recipe name is not related to the directory. The package name is biotite here and that one already exists: https://github.com/conda-forge/biotite-feedstock/blob/main/recipe/meta.yaml

> Does this mean I have to close and reopen this PR, or is there a way to _rerender_ it?

There no need to re-render on staged-recipes.

**Comment by padix-key** [comment]
Thanks for noticing it, I fixed the package name

File: recipes/biotraj/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{% set version = "1.2.0" %}
+{% set number = "0" %}
```
Comment (ocefpaf) [inline recipes/biotraj/meta.yaml:2]:
```suggestion
```

File: recipes/biotraj/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
+{% set version = "1.2.0" %}
+{% set sha256 = "d2226348546ef384d23f608cca8f7110d1e6fa8927dcbd74dbf00ee748929cd1" %}
```
Comment (ocefpaf) [inline recipes/biotraj/meta.yaml:2]:
```suggestion
```

File: recipes/biotraj/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+package:
+  name: biotraj
+  version: {{ version }}
+
+source:
+  url: https://github.com/biotite-dev/biotraj/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```
Comment (ocefpaf) [inline recipes/biotraj/meta.yaml:10]:
```suggestion
  sha256: d2226348546ef384d23f608cca8f7110d1e6fa8927dcbd74dbf00ee748929cd1
```

**Comment by padix-key** [comment]
Thanks a lot for your support @ocefpaf!

**Pull request merged**